### DISCUSSION TITLE
为什么这个命令实现不了？，亲爱的作者爸爸，首先感谢您做的这个软件。我很受用。

### DISCUSSION THREAD

**Discussion by enenurs** [shuax/MouseInc.Settings#198] [Ideas]
[
	["Execute", "C:\\Users\\Administrator\\AppData\\Local\\Microsoft\\WindowsApps\\notepad.exe"],
	["Internal", "Delay", "2000"],
	["SendKeys", "Ctrl+s"]
]

为什么，他就是不能保存。最大化窗口也不行？麻烦解答一下。感谢。。

**Comment by shuax** [comment]
SendKeys会自动激活触发窗口，如果你要发到记事本得加参数

https://docs.shuax.com/MouseInc/#/faq#%E6%80%8E%E4%B9%88%E6%88%91%E5%9C%A8clover%E4%B8%8A%E4%BD%BF%E7%94%A8%E6%89%8B%E5%8A%BF%E4%B8%8D%E8%83%BD%E5%A4%8D%E5%88%B6%EF%BC%9F

**Comment by enenurs** [comment]
[
["Execute", "C:\\Users\\Administrator\\AppData\\Local\\Microsoft\\WindowsApps\\notepad.exe"],
["Internal", "Delay", "10"],
["SendKeys", "Ctrl+s", "NOACTIVATE"],
["SendKeys", "1", "NOACTIVATE"],
["SendKeys", "Enter", "NOACTIVATE"]
]
其实我不会语言。感觉您这个软件mouseinc很好用对我来说。其实我有很多自己的想法。就害怕您不理我。我没处去问。。。还是非常感谢您能回复我邮件。


***@***.***
 
发件人： shuax
发送时间： 2021-12-01 15:14
收件人： shuax/MouseInc.Settings
抄送： enenurs; Author
主题： Re: [shuax/MouseInc.Settings] 为什么这个命令实现不了？，亲爱的作者爸爸，首先感谢您做的这个软件。我很受用。 (Discussion #198)
SendKeys会自动激活触发窗口，如果你要发到记事本得加参数
https://docs.shuax.com/MouseInc/#/faq#%E6%80%8E%E4%B9%88%E6%88%91%E5%9C%A8clover%E4%B8%8A%E4%BD%BF%E7%94%A8%E6%89%8B%E5%8A%BF%E4%B8%8D%E8%83%BD%E5%A4%8D%E5%88%B6%EF%BC%9F
―
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
